skt/__init__.py

```diff
@@ -531,15 +530,23 @@ def mktgz(self, clean=True, timeout=60 * 60 * 12):
         stdout_list = []
         self.prepare(clean)
 
-        args = self.defmakeargs + ["INSTALL_MOD_STRIP=1",
-                                   "-j%d" % multiprocessing.cpu_count(),
-                                   "targz-pkg"]
+        # Set up the arguments and options for the kernel build
+        targz_pkg_argv = [
+            "INSTALL_MOD_STRIP=1",
+            "-j%d" % multiprocessing.cpu_count(),
+            "targz-pkg"
+        ]
```

Comment:
What was wrong with the previous formatting that it needed to be changed? I won't stop the pull from being merged because of it, but it makes the definition longer without any reason (the arguments would fit on the line when starting on the same line as opening bracket).